graspologic/utils/utils.py

```diff
@@ -1030,10 +1031,17 @@ def remap_node_ids(
     if not isinstance(graph, nx.Graph):
         raise TypeError("graph must be of type nx.Graph")
 
+    if not nx.is_weighted(graph, weight=weight_attribute):
+        warnings.warn(
+            f'Graph is unweighted using weight_attribute "{weight_attribute}". Defaulting weights to "{weight_default}"'
```

Comment:
Maybe change it to:

`f'Graph has at least one unweighted edge using weight_attribute "{weight_attribute}". Defaulting unweighted edges to "{weight_default}"'`.  Then we're very precise.